kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/status/NMStatusConverter.java

```diff
@@ -57,36 +86,36 @@ public class NMStatusConverter {
 
     private static final Logger logger = LoggerFactory.getLogger(NMStatusConverter.class);
 
-    private static final byte[] DEFAULT_MAC = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 };
-
     private static final String NM_DEVICE_BUS_NAME = "org.freedesktop.NetworkManager.Device";
     private static final String NM_DEVICE_WIRELESS_BUS_NAME = "org.freedesktop.NetworkManager.Device.Wireless";
     private static final String NM_DEVICE_WIRED_BUS_NAME = "org.freedesktop.NetworkManager.Device.Wired";
     private static final String NM_ACCESSPOINT_BUS_NAME = "org.freedesktop.NetworkManager.AccessPoint";
     private static final String NM_IP4CONFIG_BUS_NAME = "org.freedesktop.NetworkManager.IP4Config";
     private static final String NM_DEVICE_PROPERTY_HW_ADDRESS = "HwAddress";
+    private static final String MM_MODEM_BUS_NAME = "org.freedesktop.ModemManager1.Modem";
+    private static final String MM_SIM_BUS_NAME = "org.freedesktop.ModemManager1.Sim";
+    private static final String MM_BEARER_BUS_NAME = "org.freedesktop.ModemManager1.Bearer";
+    private static final String MM_MODEM_LOCATION_BUS_NAME = "org.freedesktop.ModemManager1.Modem.Location";
+    private static final String MM_MODEM_3GPP_BUS_NAME = "org.freedesktop.ModemManager1.Modem.Modem3gpp";
     private static final String EMPTY_MAC_ADDRESS = "00:00:00:00:00:00";
-
-    private static final String NM_DEVICE_PROPERTY_STATE = "State";
+    private static final byte[] EMPTY_MAC_ADDRESS_BYTES = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 };
+    private static final String STATE = "State";
```

Comment:
Use the already established convention

```suggestion
    private static final String NM_DEVICE_PROPERTY_STATE = "State";
```

Reply:
I call it simply `STATE` (replacing the old one) since this constant is used both for NM and MM. So, what could be the correct name? `DEVICE_PROPERTY_STATE`? Define a `NM_DEVICE_PROPERTY_STATE` and a `MM_DEVICE_PROPERTY_STATE` with the same value :-( ?